soc/xtensa/nxp_adsp/Kconfig.defconfig

```diff
@@ -1,9 +1,29 @@
-# NXP i.MX8 SoC family default configuration options
+# NXP i.MX8/RT SoC family default configuration options
 #
 # Copyright (c) 2021 NXP
 # SPDX-License-Identifier: Apache-2.0
 
 source "soc/xtensa/nxp_adsp/*/Kconfig.defconfig.series"
 
 config CACHE_MANAGEMENT
-	def_bool y
+	default y
+
+config SMP
```

Comment:
This also seems to be disabled by default. Could probably remove this altogether from all xtensa-based NXP SoCs.